README.md

```diff
@@ -150,6 +161,10 @@ To see additional test targets (such as testing produced Docker images) use:
 make help
 ```
 
+# Contributing
```

Comment:
This section is duplicated.

Reply:
I see. Merged too early. Going to fix it.